src/vorta/log.py

```diff
@@ -13,22 +13,21 @@
 
 logger = logging.getLogger()
 
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+fh = TimedRotatingFileHandler(config.LOG_DIR / 'vorta.log', when='d', interval=1, backupCount=5)
```

Comment:
At this point `config.LOG_DIR` isn't initalized correctly yet. The following lines need to be executed first:

https://github.com/borgbase/vorta/blob/6a80518760bd1a9c77d5bc83d4b00608a99f6b8a/src/vorta/__main__.py#L63-L72